backend/src/main/resources/application.properties

```diff
@@ -7,3 +7,5 @@ logging.level.com.AZDash2=DEBUG
 jira.api.url=${JIRA_API_URL}
 jira.api.token=${JIRA_API_TOKEN}
 
+spring.jpa.properties.hibernate.ddl-auto:update
```

Comment:
Leave this in "none", because it could cause problems with local databases

It is better to give to other teams the Data Base with the new structure.